tests/__init__.py

```diff
@@ -192,6 +192,8 @@ def test_dotd(self):
                 ("odoo", "--version"),
                 # Implicit ``odoo`` command also works
                 ("--version",),
+                # Environment vars were converted to config
+                ( "custom/extras/goodconf.py", ),
```

Comment:
It seems that `odoo shell` does not autocreate a database like normal odoo commands do from v9+. Then you can change this for `("bash", "-c", "odoo --stop-after-init && custom/extras/goodconf.py")` or something similar that creates the DB before executing the shell script.